src/engraving/style/style.cpp

```diff
@@ -380,18 +406,90 @@ void MStyle::read(XmlReader& e, compat::ReadChordListHook* readChordListHook)
+            set(Sid::systemTextMinDistance, Spatium(e.readDouble()));
+        } else if (tag == "systemFrameType") { // pre-4.4 typo
+            set(Sid::systemTextFrameType, e.readInt());
+        } else if (tag == "systemFramePadding") { // pre-4.4 typo
+            set(Sid::systemTextFramePadding, e.readDouble());
+        } else if (tag == "systemFrameWidth") { // pre-4.4 typo
+            set(Sid::systemTextFrameWidth, e.readDouble());
+        } else if (tag == "systemFrameRound") { // pre-4.4 typo
+            set(Sid::systemTextFrameRound, e.readInt());
+        } else if (tag == "systemFrameFgColor") { // pre-4.4 typo
+            set(Sid::systemTextFrameFgColor, e.readColor());
+        } else if (tag == "systemFrameBgColor") { // pre-4.4 typo
+            set(Sid::systemTextFrameBgColor, e.readColor());
+        } else if (tag == "staffFontFace") { // pre-4.4 typo
+            set(Sid::systemTextFontFace, e.readText());
+        } else if (tag == "staffFontSize") { // pre-4.4 typo
+            set(Sid::staffTextFontFace, e.readDouble());
+        } else if (tag == "staffFontSpatiumDependent") { // pre-4.4 typo
+            set(Sid::staffTextFontSpatiumDependent, bool(e.readInt()));
+        } else if (tag == "staffFontStyle") { // pre-4.4 typo
+            set(Sid::staffTextFontStyle, e.readInt());
+        } else if (tag == "staffAlign") { // pre-4.4 typo
+            set(Sid::staffTextAlign, TConv::fromXml(e.readText(), Align()));
+        } else if (tag == "staffOffsetType") { // pre-4.4 typo
+            set(Sid::staffTextOffsetType, e.readInt());
+        } else if (tag == "staffPlacement") { // pre-4.4 typo
+            set(Sid::staffTextPlacement, PlacementV(e.readText().toInt()));
+        } else if (tag == "staffPosAbove") { // pre-4.4 typo
+            set(Sid::staffTextPosAbove, PointF(e.readPoint()));
+        } else if (tag == "staffPosBelow") { // pre-4.4 typo
+            set(Sid::staffTextPosBelow, PointF(e.readPoint()));
+        } else if (tag == "systemMinDistance") { // pre-4.4 typo
+            set(Sid::systemTextMinDistance, Spatium(e.readDouble()));
+        } else if (tag == "staffFrameType") { // pre-4.4 typo
+            set(Sid::staffTextFrameType, e.readInt());
+        } else if (tag == "staffFramePadding") { // pre-4.4 typo
+            set(Sid::staffTextFramePadding, e.readDouble());
+        } else if (tag == "staffFrameWidth") { // pre-4.4 typo
+            set(Sid::staffTextFrameWidth, e.readDouble());
+        } else if (tag == "staffFrameRound") { // pre-4.4 typo
+            set(Sid::staffTextFrameRound, e.readInt());
+        } else if (tag == "staffFrameFgColor") { // pre-4.4 typo
+            set(Sid::staffTextFrameFgColor, e.readColor());
+        } else if (tag == "staffFrameBgColor") { // pre-4.4 typo
+            set(Sid::staffTextFrameBgColor, e.readColor());
+        } else if (tag == "dymanicsShowTabCommon") { // pre-4.4 typo
```

Comment:
Seems to be an issue with GuitarPro tests (`XmlReader::unknown` reports this not to be recognized), not sure whether these all need mending (and where) or whether this was a typo in the code once upon a time.